client/src/resources/lang/en_us.json

```diff
@@ -513,6 +548,11 @@
     "wiki.article.standard.fullMoon.text": "Every night is a full moon night except night one and night three. The Werewolf has a different ability depending on weather or not it is a full moon night.",
     "wiki.article.standard.vision.title": "Vision",
     "wiki.article.standard.vision.text": "Visions are the investigative result that a Psychic recieves. Good visions show a list of two players that contains at least one townie. Evil visions show a list of three players that contains at least one non-townie. An Aura can effect a psychic vision.",
+    
+    "wiki.article.standard.visit.title:var.0": "Visits",
+    "wiki.article.standard.visit.title:var.1": "Visiting",
+    "wiki.article.standard.visit.title:var.2": "Visited",
+    "wiki.article.standard.visit.title:var.3": "Visitor",
```

Comment:
This is where i attempted to add more types to visit. This should work without putting it into keywords